src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,79 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: CallableFunction;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const onTabClick = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    if (event.currentTarget.hash.slice(1) === selectedTabId) {
+      return;
+    }
+
+    const currentTab = {
+      id: event.currentTarget.hash.slice(1),
+      title: event.currentTarget.textContent,
+      content: `Some text ${(event.currentTarget.textContent ?? '').replace(/^\D+/g, '')}`,
+    };
+
+    onTabSelected(currentTab);
+  };
+
+  const [activeTab, setActiveTab] = useState(tabs[0].content);
+  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
+
+  useEffect(() => {
+    const isActiveTab = tabs.find(tab => tab.id === selectedTabId)?.content
+      || tabs[0].content;
+    const isActiveTabId = tabs.find(tab => tab.id === selectedTabId)?.id
+      || tabs[0].id;
+
+    setActiveTab(isActiveTab);
+    setActiveTabId(isActiveTabId);
+  }, [selectedTabId]);
```

Comment:
simpify it. Pass tabs array as a prop, and on click chage selected tab than when you map you array if current id is an id of selectedTab show its content